packages/api/test/functional/taxonomy.spec.js

```diff
@@ -111,17 +111,11 @@ test('GET /taxonomies/:id trying to get an inexistent taxonomy', async ({ client
 	const loggeduser = await User.create(user);
 
 	const response = await client
-		.get(`/taxonomies/999`)
+		.get(`/taxonomies/99999`)
 		.loginVia(loggeduser, 'jwt')
 		.end();
 
-	response.assertStatus(400);
-	response.assertJSONSubset(
-		errorPayload(
-			errors.RESOURCE_NOT_FOUND,
-			antl('error.resource.resourceNotFound', { resource: 'Taxonomy' }),
-		),
-	);
+	response.assertStatus(204);
```

Comment:
Porquê alterar isso para 204? Se não existe, deve retornar erro 400.

Reply:
Acho que precisamos achar uma forma de fazer `findOrFail` no withParams quando for o recurso não existir.

Reply:
Na verdade basta usarmos o ``.firstOrFail` no lugar de `first` nos métodos show()